.github/workflows/cd.yaml

```diff
@@ -0,0 +1,128 @@
+name: Lanelet2 CD
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  workflow_dispatch:
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup docker buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build dependencies
+        uses: docker/build-push-action@v3
+        with:
+          builder: ${{ steps.buildx.outputs.name }}
+          tags: lanelet2_src
+          load: true
+          target: lanelet2_src
+
+      - name: Create output directory
+        run: mkdir -p ${{ github.workspace }}/output
+
+      - name: Do stuff inside the container
+        uses: addnab/docker-run-action@v3
+        with:
+          image: lanelet2_src
+          shell: bash
+          options: -v ${{ github.workspace }}/dist:/dist
+          run: |
```

Comment:
Theoretically you can change the python version used by CMake in 
`catkin config --cmake-args -DCMAKE_BUILD_TYPE=Release -DPYTHON_VERSION=$PYTHON_VERSION` (line 48)
with ` -DPYTHON_VERSION`, but when I tried this I ran into problems with `boost_python`, as `boost_python` is only working with the system python version.

From what I gathered, the options are:

1. Compile boost from source in the CD for each additional python version we want to ship
2. Do the wheel building process in ROS2 again for a python 3.10 wheel. This means however that we could only get wheels for 3.8 and 3.10

I tend to agree though that this may be better tackled in a follow-up PR :sweat_smile:  
In this case, we should probably remove the tests for the unsupported python versions from the CD for now